Makefile

```diff
@@ -4,6 +4,7 @@ REAL_ENV = dev
 NAME = storybook-${REAL_ENV}
 TAG = libero/storybook:${REAL_ENV}
 EXISTING_CONTAINERS = $$(docker ps --all --quiet --filter "name=${NAME}")
+MOUNT := $$([[ ${REAL_ENV} = "dev" ]] && echo "--mount type=bind,source=$$(pwd)/src/,destination=/app/src,readonly" || echo "")
```

Comment:
could use a `if`, as a failure in the `&& ...` then statement jumps to the else:
```
$ [[ "dev" == "dev" ]] && echo-typo 1 || echo 2
echo-typo: command not found
2
```